src/component/contents/DraftEditorContents-core.react.js

```diff
@@ -192,7 +192,6 @@ class DraftEditorContents extends React.Component<Props> {
         preventScroll,
         selection,
         tree: editorState.getBlockTree(key),
-        key: `${key}-${blockKeyMap.get(key) || '0'}`,
```

Comment:
Will the key removal the parent block of the `<Component />` ?

Reply:
No, it won't.
React uses key to identify the existing element which won't be unmounted. Also, an element and its descendants won't be unmounted only if its type (e.g. div, span) and key keep the same with the previous one. If an element with no key, React only compares its type and index(index in a children list).

Since <Component /> is the only child of its parent [`Child`](https://github.com/twreporter/draft-js/pull/2/commits/a8860c52628d5d22d0515f5c610d5f18ba8b1dd8#diff-f57ffae9860a2a40860ba9423c0241282b39f889a8f201a053037499a9b8078cL243-L247), this diff removes the key from here to its parent which I don't think has any side-effect, AFAIK.